src/errors.rs

```diff
@@ -26,7 +26,10 @@ quick_error! {
         SendError(msg: String) {
             display("Push message could not be sent: {}", msg)
         }
-        ProcessingError(msg: String) {
+        ProcessingServerError(msg: String) {  // Caused by server. Retrying might help.
```

Comment:
What perspective are we talking about here: Who is the server and who is the client? Perhaps we can avoid the terms *server* and *client* if it would avoid confusion.

Reply:
With "server" I meant the push server (GCM / APNs). Do you have a proposal for better terminology?

Reply:
*remote server* or just *remote*? :slightly_smiling_face:

Reply:
Sounds good :)